tests/run.html

```diff
@@ -3,7 +3,7 @@
 <head>
 	<meta charset="utf-8">
 	<title>Intern suite</title>
-	<meta http-equiv="refresh" content="0;url=../node_modules/intern/client.html?config=_build/tests/intern">
+	<meta http-equiv="refresh" content="0;url=../../node_modules/intern/index.html">
```

Comment:
do you need the extra `..`?

Reply:
I guess it depends where you serve from. I ran my http server from the project root and this works correctly.

Reply:
it's got to step out of `_build/tests` to get to the `node_modules` folder, so yes, double `../..` required.

Reply:
Are you running from the `_build` directory, because I think now you can just run from the `tests` directory

Reply:
thanks, that works